test/unit/indentation_test.exs

```diff
@@ -0,0 +1,180 @@
+defmodule ExFormat.Unit.IndentationTest do
+  import Test.Support.Unit
+  use ExUnit.Case
+
+  describe "pipeline indentation" do
+    test "do not split if pipeline has no line break" do
+      assert_format_string("1 |> 2 |> 3 |> 4\n")
+    end
+
+    test "split if pipeline is too long even though it has no line break" do
+      bad = "input |> Some.verylongcall() |> Again.verylonglongcall() |> AgainAgain.verylonglongcall() |> AgainAgainAgain.verylonglongcall"
+      good = """
+      input |> Some.verylongcall() |> Again.verylonglongcall()
+      |> AgainAgain.verylonglongcall()
+      |> AgainAgainAgain.verylonglongcall()
```

Comment:
Yes, I agree. Will revisit this soon since splitting all pipes involves quite a bit work.